app/services/nomis/client.rb

```diff
@@ -31,14 +31,24 @@ def get(route, queryparams: {}, extra_headers: {})
       data
     end
 
+    def post(route, body)
+      response = request(
+        :post, route, body: body, extra_headers: {
+          'Content-Type' => 'application/json'
+        }
+      )
+      JSON.parse(response.body)
+    end
+
   private
 
-    def request(method, route, queryparams: {}, extra_headers: {})
+    def request(method, route, body: nil, queryparams: {}, extra_headers: {})
```

Comment:
Should the body default be an empty hash? `{}`

Reply:
I'm not sure.  It could just as arguably be an empty list, but as both that and {} are valid json I think nil might be a better choice.